app/src/main/java/com/noalecohen/dispatcher/customview/CustomHeader.kt

```diff
@@ -0,0 +1,47 @@
+package com.noalecohen.dispatcher.customview
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import android.widget.ImageButton
+import android.widget.ImageView
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.view.isVisible
+import com.noalecohen.dispatcher.R
+import com.noalecohen.dispatcher.databinding.CustomHeaderBinding
+
+class CustomHeader(context: Context, attrs: AttributeSet?) : ConstraintLayout(context, attrs) {
+
+    var binding: CustomHeaderBinding
+    var firstIcon: ImageView
+    var secondIcon: ImageButton
+    var thirdIcon: ImageButton
+    var view: View = inflate(context, R.layout.custom_header, this)
+
+    init {
+        binding = CustomHeaderBinding.bind(view)
+        firstIcon = binding.headerFirstIcon
+        secondIcon = binding.headerSecondIcon
+        thirdIcon = binding.headerThirdIcon
+    }
+
+    fun isShowIcon(icon: ImageButton): Boolean {
+        return icon.isVisible
+    }
+
+    fun setShowIcon(icon: View, toShow: Boolean) {
+        if (toShow) {
```

Comment:
this could be written in one line like so
if(condition) something else somethingElse

Reply:
also with the assignment.
i.e: 
`binding.icon.visibiliity = if (toShow) View.VISIBLE else View.GONE`